matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPhenotypesMap.java

```diff
@@ -219,17 +219,29 @@ public static void reorder(List<Map<String, String>> predefined1, List<Map<Strin
             if (StringUtils.equals(name1, name2)) {
                 common1.add(item1);
```

Comment:
just for the sake of future developers, i would extend the method comment and mention that the incoming lists (`predefined1` and `predefined2`) are supposed to be sorted, as otherwise the algorithm does not work